src/App.jsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import './App.css'
+import play from './access/paly.svg'
+import pause from './access/pause.svg'
+import resetTime from './access/time.svg'
+import arrow from './access/arrow.svg'
+import {connect} from "react-redux";
+import {addBreakLength, addSessionLength, reset, startStopTimer} from "./redux/appReduxer";
+import {getFormate} from "./utils";
+
+let Setting = ({title, value, upWork, upRest, downWork, downRest, category}) => {
+  let onUpArrow = () => {
+    category === 'work' ? upWork() : upRest();
+  }
+  
+  let onDownArrow = () => {
+    category === 'work' ? downWork() : downRest();
+  }
+  return (
+    <div className='setting'>
+      <div className='settingTitle'>{title}</div>
+      <div className='withArrows'>
+        <img src={arrow} alt="upValue" onClick={onUpArrow} className="upValue"/>
+        <div className='settingValue'>{value}</div>
+        <img src={arrow} alt="downValue" onClick={onDownArrow} className="downValue"/>
+      </div>
+    </div>
+  )
+}
+
+let ButtonWithSvg = ({svg, alt, onClick}) => {
+  return <img className='buttonImg' onClick={onClick || null} src={svg} alt={alt}/>
+}
+
+class App extends React.Component {
```

Comment:
You have a new code base, be consistent in types of components. Use either functional components or class components

Reply:
Converted to Functional Component